Tools-Override/tests.targets

```diff
@@ -198,6 +198,13 @@
       Overwrite="true"
       Encoding="Ascii" />
 
+      <!--======================================================
+                      Section for netfx test runs
+            ====================================================== -->
+    <ItemGroup Condition="'$(BuildingNETFxVertical)' == 'true'" >
+      <TestCommandLines Include="set DEVPATH=%RUNTIME_PATH%" />
```

Comment:
If we are setting this we also should call "setlocal" assuming we aren't already in this batch file, to ensure this property remains scoped to just this one call.